endpoints.json

```diff
@@ -30,5 +30,114 @@
         }
       ]
     }
+  },
+  "GET /api/reviews/:review_id": {
+    "description": "Serves a single review by review_id.",
+    "queries": [],
+    "exampleResponse": {
+      "review": {
+        "review_id": 2,
+        "title": "Jenga",
+        "review_body": "Fiddly fun for all the family",
+        "designer": "Leslie Scott",
+        "review_img_url": "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+        "votes": 5,
+        "category": "dexterity",
+        "owner": "philippaclaire9",
+        "created_at": "2021-01-18T10:01:41.251Z",
+        "comment_count": 3
+      }
+    }
+  },
+  "PATCH /api/reviews/:review_id": {
+    "description": "Patches a single review vote by review_id.",
+    "queries": [],
```

Comment:
You'll thank me in 2.5 weeks time, but I would add an exampleRequest i.e. {inc_votes: 1}

Otherwise someone using the api doesnt know how stuff needs to be sent. 
Also, you wont have to check back in your backend when you start connecting up your front end in a few weeks time